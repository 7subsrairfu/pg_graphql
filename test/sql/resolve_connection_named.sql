select gql.resolve(
    $$
    query FirstNAccounts($first: Int!) {
      allAccounts(first: $first) {
        edges {
          node {
            id
          }
        }
      }
    }
    $$,
    '{"first": 2}'::jsonb
);

