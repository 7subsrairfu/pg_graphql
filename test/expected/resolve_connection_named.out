select gql.dispatch(
    $$
    query FirstNAccounts($first: Int!) {
      allAccounts(first: $first) {
        edges {
          node {
            id
          }
        }
      }
    }
    $$,
    '{"first": 2}'::jsonb
);
                                            dispatch                                            
------------------------------------------------------------------------------------------------
 {"data": {"allAccounts": {"edges": [{"node": {"id": 1}}, {"node": {"id": 2}}]}}, "errors": []}
(1 row)

